Vivarium/Assets/Scripts/Player/PlayerController.cs

```diff
@@ -112,6 +112,8 @@ private void SelectAction(Action action)
         var actionAOE = StatCalculator.CalculateStat(action, StatType.AttackAOE);
         _selectedActionRange = StatCalculator.CalculateStat(action, StatType.AttackRange);
         attackViewer.DisplayAction(actionAOE, _selectedActionRange);
+        UIController.Instance.MoveRangeText.text = "R: " + _selectedAction.Range.ToString("N0");
+        UIController.Instance.DamageText.text = "D: " + _selectedAction.BaseDamage.ToString("N0");
```

Comment:
I would move this logic into the `UIController`. Basically, create a `DisplayActionStats` method and call that hear. So, it would look something like this:
```
UIController.Instance.DisplayActionStats(_selectedAction);
```
The reason I'm asking this is because it is generally a good practice to separate unrelated logic. In other words, `PlayerController` should only have to worry about handling player controls while `UIController` should worry about displaying the correct UI.